ops/manifests/manifest-cb.yaml

```diff
@@ -0,0 +1,32 @@
+---
+applications:
+- name: getgov-cb
+  buildpacks:
+    - python_buildpack
+  path: ../../src
+  instances: 1
+  memory: 512M
+  stack: cflinuxfs4
+  timeout: 180
+  command: ./run.sh
+  health-check-type: http
+  health-check-http-endpoint: /health
+  health-check-invocation-timeout: 40
```

Comment:
This line is new as compared to https://github.com/cisagov/manage.get.gov/pull/859/files

Reply:
Looked and we added that later so all manifests now have this line